plugins/parcelize/parcelize-runtime/build.gradle.kts

```diff
@@ -7,7 +7,6 @@ plugins {
 
 dependencies {
     api(kotlinStdlib())
-    api(project(":kotlin-android-extensions-runtime"))
```

Comment:
I wonder if this is safe? In theory someone could've used exported symbols from AE runtime through parcelize-runtime...

Reply:
There's this issue: https://youtrack.jetbrains.com/issue/KT-43538/kotlin-parcelize-runtime-should-not-expose-kotlin-android-extensions-runtime-dependency-or-remove-it-if-its-not-necessary

Reply:
That is a good point. All parcelize tests pass, but if someone ends up using AE runtime symbols directly without going through parcelize codegen it could be a problem.

What that means is probably that we should take this a bit slower than what I'm doing here. We can take this in multiple steps (version suggestions are only suggestions):

    - make android extensions compiler plugin report a compilation error (1.8.0)
    - remove the publishing of the android extensions plugin (1.8.20)
    - remove the android synthetics runtime dependency from parcelize runtime (1.8.20)
    - remove the code (1.9.0)

If that sounds OK to you I'll get step 1 done today and then start the next couple of steps after 1.8.0 branches.

Reply:
Yes, it sounds fine to me!